docs/plugins/unimported.rst

```diff
@@ -0,0 +1,18 @@
+Play Plugin
+===========
+
+The ``unimported`` plugin allows to list all files in the library folder which are not imported.
```

Comment:
Maybe it would be clearer to say "which are not listed in the beets library database" or something?